modules/vmfcore/include/vmf/metadatastream.hpp

```diff
@@ -53,15 +54,16 @@ class VMF_EXPORT MetadataStream : public IQuery
 {
 public:
     /*!
-    * \brief File open mode enumeration
+    * \brief File open mode flags
     */
-    enum OpenMode
+    enum OpenModeFlags
     {
-        InMemory, /**< Stream data are in-memory */
-        ReadOnly, /**< Open file for read only */
-        ReadWrite, /**< Open file for read and write */
-
+        InMemory  = 0, /**< Stream data are in-memory */
+        ReadOnly  = 1, /**< Open file for read only */
+        ReadWrite = 2, /**< Open file for read and write */
```

Comment:
please rename `ReadWrite` to `Update` to be consistent with XMP open mode flags.